internal/cypress/config.go

```diff
@@ -23,6 +23,7 @@ type Project struct {
 	BeforeExec     []string           `yaml:"beforeExec,omitempty" json:"beforeExec"`
 	Docker         config.Docker      `yaml:"docker,omitempty" json:"docker"`
 	Npm            config.Npm         `yaml:"npm,omitempty" json:"npm"`
+	RootDir        string             `yaml:"rootDir,omitempty" json:"rootDir"`
```

Comment:
Maybe I recall wrongly, but I remember this would be introduced also for Playwright and TestCafe (and take precedence over `ProjectPath` those 2 frameworks).

Reply:
I guess we will rename `ProjectPath` to `RootDir` for Playwright and TestCafe.

Reply:
Renaming it would make it backwards incompatible. We'll need to keep it for a while and have a fallback for playwright and testcafe, with a deprecation warning message if people still use it.

Reply:
I created a separate task for this https://saucedev.atlassian.net/browse/DEVX-687 .